### ISSUE TITLE
move TestData outside of the source tree

### ISSUE THREAD

**Issue by takanokage** [IntelVCL/Open3D#378]
Move `<Open3D Path>/src/Test/TestData` and update all the Python scripts, C++ code and documentation.

TestData could be placed instead at the same level with `src` like this:

- `<Open3D Path>/src/Test`
- `<Open3D Path>/TestData`

TestData is over 70MB of data only files and could grow in the future.

**Comment by syncle** [comment]
I agree with this change. Let's do this once cmake install is merged.

**Comment by jingpengw** [comment]
yep, currently the TestData place do not work with the python tutorials.
